store/server/features/deals/mutations.ts

```
@@ -0,0 +1,117 @@
+              } else {
+                formData.append(key, value);
+              }
+            }
+          }
+        });
+        
+        // Add the file
+        formData.append('dealDocument', dealData.dealDocument);
+        
+        // Remove Content-Type header to let browser set it with boundary for FormData
+        const { 'Content-Type': _, ...headersWithoutContentType } = headers as any;
```

ESLint

[error] '_' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)